draft-patton-cfrg-vdaf.md

```diff
@@ -705,6 +705,147 @@ TODO
 * Output shares are secret shares of a vector of field elements, each
   corresponding to a counter for one of the candidate prefixes.
 
+## Dependencies
+
+[TODO Clarify which of these can be assumed and which need to be explicitly defined]
+
+### Descriptions of finite fields
```

Comment:
Add a newline between headings and the first line.

Reply:
Done.